docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/01_instrument_intro_prerequisites.md

```diff
@@ -0,0 +1,57 @@
+## Introduction to Instruments
+
+The Behavioural Database is the part of LORIS that implements the battery of tests and questionnaires which are administered and used to collect data on participants in a given study. These behavioural, psychological, and clinical tests are known as **instruments** or instrument forms.  
+
+Basic instrument forms can be complemented with automated **scoring** and **validation**. The implementation of scoring and validation varies based on the chosen instrument format, the instructions below describe in more details how the implementation should be done for each type.
+
+The **battery** is a list of which instruments are to be administered on each cohort of participants, and typically varies per timepoint and per study site.  For example, the battery of instruments administered at a participant's first visit (e.g. timepoint V01) may depend on which cohort and study site they belong to.  A participant may be tested on a certain set of instruments if they are in cohort A and from site X, but would be tested on different instruments if they are registered in cohort B at site Y (For more information about Battery assignments, navigate directly to the [Clinical Test Battery](./07_clinical_test_battery.md) section or continue following this order of the).
+
+### How are Instruments Created?
+
+LORIS instruments can be implemented in one of 2 formats: **LINST** or **PHP**.
+
+**LINST** is a LORIS in-house form format which is generated and edited by the LORIS Instrument Builder module. The LINST format allows for the creation of basic forms but can be complemented by a *score* file to allow for automated scoring and a *rules* file to accommodate for automatically checked rules.
```

Comment:
```suggestion
**LINST** is a custom LORIS form format which is generated and edited by the LORIS Instrument Builder module. The LINST format allows for the creation of basic forms with limited functionality and can be complemented by a *score* file to allow for automated scoring and a *rules* file to accommodate for validation of dependencies between form questions and answers.
```
- changing in-house to "custom" (i think in-house won't make as much sense to externals)
- changing "rules" wording to match the very nice intro on this file which emphasizes their purpose: validation

Reply:
will commit with a slight mod. I removed the `limited functionality` as it sounds pejorative here